riffdog/scanner.py

```diff
@@ -28,7 +30,9 @@ def scan():
 
     _load_resource_modules()
 
-    # Scan current repo
+    if not rd.found_resources:
+        print(NO_FOUND_RESOURCES_ERROR)
```

Comment:
Ok, so I was super interested to see what you were going to do here.

Basically I don't think we can do this (`system.exit(0)`), because scanner is a library not just command line - I think we have to do 

`raise PickFavException(NO_FOUND_RESOURCES_ERROR)` 

and then have the command line capture that and print it off (when in command line mode), Thoughts?

Reply:
(or at least, return null at this point? )

Reply:
@jmons yeah that occurred to me after I pushed this so now:

`command_line.py` can `sys.exit` but everywhere else raises exceptions that are handled in `command_line.main`.

Reply:
Also, errors now go to stderr